packages/zowe-explorer/__tests__/__unit__/utils/SpoolUtils.unit.test.ts

```diff
@@ -101,7 +101,11 @@ describe("SpoolProvider Unit Tests", () => {
             bindJesApi(jesApi);
 
             const spoolOk: zosjobs.IJobFile = { ...iJobFile, stepname: "test", ddname: "dd", "record-count": 1, procstep: "proc" };
-            const { id, ddname, stepname, ...withoutIdDdStep } = spoolOk;
+            // const { id, ddname, stepname, ...withoutIdDdStep } = spoolOk;
+            const withoutIdDdStep: Partial<zosjobs.IJobFile> = JSON.parse(JSON.stringify(spoolOk));
+            delete withoutIdDdStep.id;
+            delete withoutIdDdStep.ddname;
+            delete withoutIdDdStep.stepname;
```

Comment:
I assume this change was made because of linter complaining about unused variables. Curious if the following would work:
```suggestion
            const { _id, _ddname, _stepname, ...withoutIdDdStep } = spoolOk;
```

Reply:
I'm afraid it would not work since `_id`, `_ddname`, and `_stepname` are not part of the `zosjobs.IJobFile` interface.

We could 
`const { id: _id, ddname: _ddname, stepname: _stepname, ...withoutIdDdStep } = spoolOk;`
But that still triggers the lint warning
<img width="701" alt="image" src="https://github.com/user-attachments/assets/31cf900e-2879-4201-a50e-b07cb3f9e71f">